api/app/events/api.py

```diff
@@ -388,12 +366,21 @@ def post(self):
             firstname = user.firstname
             lastname = user.lastname
             event_name = event.name
+            organisation_name = event.organisation.name
+            system_name = event.organisation.name
```

Comment:
There is a system_name property on organisation :)

Reply:
Oh yes I saw that 🙈 Think I was on autopilot. Good catch!